shared/constants/fs.js

```diff
@@ -4,6 +4,7 @@ import * as Types from './types/fs'
 import uuidv1 from 'uuid/v1'
 import {globalColors} from '../styles'
 import {downloadFilePath} from '../util/file'
+import {shell} from 'electron'
```

Comment:
do not directly import things from electron in our shared code. this won't work in rn and will break the build

Reply:
Yes will do; I did this temporary thing just to make it work. Will rebase after John's changes are in. Thanks!